pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -1,9 +1,10 @@
 { lib
+, pkgs
 , fetchpatch
 , kernel
-, date ? "2022-04-25"
-, commit ? "bdf6d7c1350497bc7b0be6027a51d9330645672d"
-, diffHash ? "09bcbklvfj9i9czjdpix2iz7fvjksmavaljx8l92ay1i9fapjmhc"
+, date ? "2022-07-25"
+, commit ? "5549e504645ac9ae570323eab6417b560a7def9b"
+, diffHash ? "sha256-DStuK3TSFB2BgVLMFxhL+q7RteURvYDj/RF14D5ENuw="
```

Comment:
Package arguments should not be non-package arguments unless there is little chance packages with that name will be introduced (like `enableFoo` or `withBar`). Otherwise `callPackage` will substitute them when e.g. #172568 is merged.